client/src/main/java/io/split/client/LocalhostSplitClient.java

```diff
@@ -20,47 +21,75 @@
  */
 public final class LocalhostSplitClient implements SplitClient {
     private static final Logger _log = LoggerFactory.getLogger(LocalhostSplitClient.class);
+    private static SplitResult SPLIT_RESULT_CONTROL = new SplitResult(Treatments.CONTROL, null);
```

Comment:
When a configuration is not defined, should we return an empty map instead of null? It would simplify / remove NPEs